project/plugins.sbt

```diff
@@ -1,15 +1,7 @@
-credentials += Credentials(Path.userHome / ".sbt" / ".credentials")
+resolvers += Resolver.url("hmrc-sbt-plugin-releases",
+  url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
```

Comment:
Previously I have also used:

```
resolvers ++= Seq(
  Resolver.bintrayRepo("hmrc", "releases"),
  "typesafe-releases" at "https://nexus-preview.tax.service.gov.uk/content/repositories/typesafe-releases/"
)
```

Are the various resolvers documented anywhere so I can see what of coming from the various places?